.gitignore

```diff
@@ -57,3 +57,7 @@ build/Release
 /pnpm-publish-summary.json
 /tests/smoke-tests/deployment
 /packages/foundation-models/src/azure-openai/azure-openai-types-schema.ts-e
```

Comment:
[nth] Potentially this also belongs into the zod bucket?

Reply:
Oh I missed this file. It is no longer needed. Removed.